server/moodle/mod/livequiz/tests/lib_test.php

```diff
@@ -20,7 +20,7 @@
  * @copyright 2023
  * @license http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
-namespace mod_livequiz;
+
```

Comment:
why was the namespace removed?

Reply:
had to remove it cause it crashes the tests, dont know why it has been pushed in the first place since the tests is not running in the staging branch

Reply:
> had to remove it cause it crashes the tests, dont know why it has been pushed in the first place since the tests is not running in the staging branch

Code sniffer puts it as a requirement, remember to fix all codeaniffer issues before we can properly review

Reply:
fair enough i guess, although i think we're supposed to still have it no?